docs/intake-api.asciidoc

```diff
@@ -21,16 +18,16 @@ The request body looks roughly like this:
 
 With NDJSON, agents can open an HTTP POST request and use chunked encoding to stream events to the APM Server as soon as they are recorded in the agent.
 This makes it simple for agents to serialize each event to a stream of newline delimited JSON.
-
 The APM Server also treats the HTTP body as a compressed stream and thus reads and handles each event independently.
```

Comment:
All of the above description is part of the events Intake API and should be moved.